core-rust/core-api-server/src/core_api/generated/models/substate.rs

```diff
@@ -13,6 +13,10 @@
 #[derive(Clone, Debug, PartialEq, serde::Serialize, serde::Deserialize)]
 #[serde(tag = "substate_type")]
 pub enum Substate {
+    #[serde(rename="Account")]
+    AccountSubstate {
+        // TODO(code review): what do we return?
```

Comment:
This should be generated by updating the `core-api-schema.yaml` file and then running:
```
python3 core-rust/core-api-server/scripts/generate-openapi-server.py
```

I would suggest following the same pattern used by AccessController.

Reply:
> This should be generated

Huh, I guess the `/generated/` in the path is not enough of a hint for me 🤷‍♂️ 

> following the same pattern used by AccessController

I interpret this as "literally return the same `data_struct`" (which reveals I have no clue about that api) 👍